pom.xml

```diff
@@ -32,9 +32,9 @@ under the License.
 	<artifactId>flink-parent</artifactId>
 	<version>0.9-SNAPSHOT</version>
 
-	<name>flink</name>
+	<name>Apache Flink</name>
```

Comment:
I vote to undo this change. This is the name displayed in maven (command line and more) and not just some "pretty id" as I thought.